examples/kotlin-diktat/sarif-actual-expected/save-warnings-actual.sarif

```diff
@@ -0,0 +1,71 @@
+{
+  "$schema": "https://raw.githubusercontent.com/oasis-tcs/sarif-spec/master/Schemata/sarif-schema-2.1.0.json",
+  "version": "2.1.0",
+  "runs": [
+    {
+      "originalUriBaseIds": {
+        "%SRCROOT%": {
+          "uri": "file://D:/projects/"
+        }
+      },
+      "results": [
+        {
+          "level": "error",
+          "locations": [
+            {
+              "physicalLocation": {
+                "artifactLocation": {
+                  "uri": "src\\kotlin\\EnumValueSnakeCaseTest.kt",
+                  "uriBaseId": "%SRCROOT%"
+                },
+                "region": {
+                  "startColumn": 5,
+                  "startLine": 18
+                }
+              }
+            }
+          ],
+          "message": {
+            "text": "[ENUM_VALUE] enum values should be in selected UPPER_CASE snake/PascalCase format: NAme_MYa_sayR_"
+          },
+          "ruleId": "diktat-ruleset:identifier-naming"
+        },
+        {
+          "level": "error",
+          "locations": [
+            {
+              "physicalLocation": {
+                "artifactLocation": {
+                  "uri": "src\\kotlin\\GenericFunctionTest.kt",
```

Comment:
Do we use it?

Reply:
`EnumValueSnakeCaseTest` may be?

Reply:
Yes. Removed it

Reply:
Usually `locations` is additionally set to indicate with which files do we operate, so, generally more appropriate way was just to rename the `GenericFunctionTest` to `EnumValueSnakeCaseTest`
However, it's not required part in SARIF, so we can remove it too, okay